styles/media-queries.css

```diff
@@ -2,26 +2,93 @@
 /*  */
 /*}*/
 
-/*@media (max-width: 1280px) {*/
+@media (max-width: 1280px) {
+
+}
+
+/*@media (max-width: 1024px) {*/
 
 /*}*/
 
-@media (max-width: 1024px) {
-  .footer__list {
-    width: 150px;
+@media (max-width: 768px) {
+  .logo__header {
+    width: 122px;
+    height: 24px;
   }
-}
 
-@media (max-width: 768px) {
-  .footer__list {
-    justify-content: center;
+  .advantages {
+    padding-top: 60px;
+    padding-bottom: 61px;
+  }
+
+  .footer {
+    padding-top: 60px;
+    padding-bottom: 60px;
   }
+
+
 }
 
 @media (max-width: 480px) {
+  .advantages {
+    padding: 24px 73px 40px 73px;
+  }
 
-}
+  .advantages__header {
+    font-size: 28px;
+    line-height: 33px;
+    margin-bottom: 32px;
+  }
+
+  .advantages__highlight {
+    display: none;
+  }
+
+  .advantages__list {
+    row-gap: 32px;
+  }
+
+  .card__icon {
+    width: 70px;
+    height: 70px;
+    margin-bottom: 16px;
+  }
+
+  .card__text {
+    font-size: 15px;
+    line-height: 18px;
+  }
+
+  .footer {
+    padding: 40px 18.88% 40px 18.88%;
+  }
+
+  .footer .logo {
+    display: flex;
+  }
 
-@media (max-width: 360px) {
+  .footer__menu {
```

Comment:
Your footer design not looks like on figma file. Please, rewrite it accordingly with design

Reply:
I can't find where the footer looks wrong in my layout